src/components/layouts/Footer.tsx

```diff
@@ -0,0 +1,180 @@
+import { Flex, Link, Stack, Text } from '@chakra-ui/react';
+import { FC } from 'react';
+import NextLink from 'next/link';
+
+import {
+  DOCS_PAGE,
+  DOWNLOADS_PAGE,
+  GETH_DISCORD_URL,
+  GETH_REPO_URL,
+  GETH_TWITTER_URL
+} from '../../constants'
+
+import {
+  DiscordIcon,
+  GitHubIcon,
+  TwitterIcon
+} from '../UI/icons';
+
+export const Footer: FC = () => {
+  return (
+    <Flex
+      mt={4}
+      direction={{ base: 'column', lg: 'row'}}
+    >
+      <Flex
+        direction={{ base: 'column', md: 'row'}}
+        justifyContent={{md: 'space-between'}}
+        border='2px solid'
+        borderColor='brand.light.primary'
+      >
+        <Flex
+          sx={{ mt: '-2px !important' }}
```

Comment:
Do we need this? I have the feeling that this layout could be done using common style props.